### PR TITLE
Add Vetro theme

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR adds a new theme designed by @henriqueiamarino 🎉 

I have cleaned up the export quite a bit, so please let me know if I've mistakenly removed anything that was needed.

There are a couple remaining things that should be done / audited before this is shipped: 

- [x] Remove the alignment CSS in favor of core's global padding support
- [x] Migrate all text to patterns and escape + translate it (home, 404, single) — is this still a requirement?

### CODE REVIEW

File: vetro/theme.json [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+				"line-height": {
+					"medium": 1.4,
+					"normal": 1.6,
+					"small": 1.2,
+					"tiny": 1.15
+				}
+			}
+		},
+		"layout": {
+			"contentSize": "1000px",
```
Comment (jffng) [inline vetro/theme.json:40]: Should the `contentSize` be narrower than the `wideSize`? @henriqueiamarino

File: vetro/theme.json [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+		"custom": {
+			"spacing": {
+				"large": "clamp(4rem, 10vw, 8rem)",
+				"medium": "clamp(2rem, 8vw, calc(4 * var(--wp--style--block-gap)))",
+				"outer": "var(--wp--custom--spacing--small, 1.25rem)",
+				"small": "max(1.25rem, 5vw)"
+			},
+			"typography": {
+				"font-size": {
+					"colossal": "clamp(3.25rem, 8vw, 6.25rem)",
```
Comment (jffng) [inline vetro/theme.json]: Are the `colossal` and `huge` custom font sizes used anywhere?

File: vetro/readme.txt [hunk 3]
```diff
@@ -0,0 +1,31 @@
+=== Vetro ===
+Contributors: Dotorg Themes Design
```
Comment (jffng) [inline vetro/readme.txt]: Probably just automattic here:

```suggestion
Contributors: Automattic
```

File: vetro/readme.txt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Requires at least: 5.8
+Tested up to: 5.9
```
Comment (jffng) [inline vetro/readme.txt]:
```suggestion
Tested up to: 6.1
```

File: vetro/readme.txt [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Vetro was designed to be a portfolio theme, and is composed by wide width layouts with the impact of generous imagery and typography. Its simple pages are aligned left with fair right paddings and large content blocks to grant viewers focus on visuals and short paragraphs.
+
+== Changelog ==
+
+= 0.0.1 =
+* Initial release
+
+== Copyright ==
+
+Vetro WordPress Theme, (C) 2022 Dotorg Themes Design
```
Comment (jffng) [inline vetro/readme.txt]:
```suggestion
Vetro WordPress Theme, (C) 2022 Automattic
```

File: vetro/style.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
+/*
+Theme Name: Vetro
+Theme URI: https://vetro.mystagingwebsite.com/
```
Comment (jffng) [inline vetro/style.css:3]:
```suggestion
Theme URI: https://wordpress.com/theme/vetro
```

File: vetro/style.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+Author: Dotorg Themes Design
+Author URI: https://wordpress.org/themes/
```
Comment (jffng) [inline vetro/style.css:5]:
```suggestion
Author URI: https://automattic.com/
```

File: vetro/templates/index.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```
Comment (jffng) [inline vetro/templates/index.html:1]: Noting that with the templates this theme currently supplies, `index` would never be used.

**Comment by henriqueiamarino** [comment]
Hey, @jffng I have populated the [test site](https://vetrodemo.wordpress.com/) as required and have 3 notes on it.

**Group block alignment in Home**
It's working nicely in the Editor but in Frontend it gets scruffy. 

vetro-review_1home


**Padding on pages**
As checked on this PR I thought this has been fixed. Not sure if I should report it but it's here just in case...

vetro-review_2page


**Right Padding on the content blocks**
This makes a lot of sense to me on the Desktop version but I'd like to have it removed on Mobile. Let's keep Right and Left padding even on small screens if possible

vetro-review_3content

**Pull request merged**